certified-connectors/DeepL/readme.md

```diff
@@ -0,0 +1,19 @@
+# DeepL API
+Our goal is to overcome language barriers and bring cultures closer together. In order to achieve that goal, 
+we are using our expertise in artificial intelligence and neural networks to create technologies that make communication faster, better, and easier.
+
+The DeepL API provides programmatic access to DeepL’s machine translation technology, making it possible to bring high quality translation capabilities directly to your websites and applications.
+
```

Comment:
@DeeJayTC please include all required section for the readme file. please follow this [template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md).
Please add the following.
```
# Publisher: DeepL
# Supported Operations (please list some of them, then you can refer for the rest operation using the external documentation)
# Obtaining Credentials
# Known Issues and Limitations
# Deployment Instructions
```